### DISCUSSION TITLE
Sequel ORM support

### DISCUSSION THREAD

**Discussion by evan-paul** [excid3/noticed#347] [Q&A] [answered]
Taking a look at the model generator implies a dependency of ActiveRecord. I'm currently using [Sequel ORM;](https://sequel.jeremyevans.net/) is using noticed out of the question or will I be okay if I workaround these generators?

**Comment by excid3** [comment, marked as answer]
I don't have any plans to support other ORMs right now.

**Reply by evan-paul** [reply]
Thanks for the quick response. I may make an attempt at forking this and seeing if I can make it work with Sequel ORM. If it were generalized enough so that ActiveRecord support wasn't removed by this change, would this be something you would consider merging in (assuming it's up to your standards) or do you prefer that this library remains purely based on ActiveRecord?

I'm asking because generalizing it would be more work and so if you're not interested in that I won't bother.

**Reply by excid3** [reply]
Open to it, but I think it will get tricky to maintain since we're using ActiveRecord functionality and would have to duplicate all that work going forward. But I don't know Sequel well enough, so it might be easier than I think.
